app/assets/v2/css/retail/tribes_landing.css

```diff
@@ -47,53 +38,102 @@ img {
   top: 1px;
 }
 
+.plan {
+  min-height: 3rem;
+}
+
 .plan-header img {
   height: 120px;
 }
 
-.testimonial-section {
-  background: #000;
+.why-section {
+  background: #170640;
   color: #fff;
 }
 
-.testimonial-section .square-1 {
-  top: 2rem;
+.why-section .square-1 {
+  top: -9rem;
   right: 0;
+  opacity: 0.2;
 }
 
-.testimonial-section .square-2 {
+.why-section .square-2 {
   height: 25rem;
+  top: 25rem;
+  opacity: 0.3;
+}
+
+.why-section .reasons img {
+  height: 2.2rem;
+}
+
+.zoom-on-hover {
+  transition: all .3s;
+}
+
+.zoom-on-hover:hover {
+  transform: scale(1.15);
 }
 
-.testimonial-section .stats {
-  font-size: 3rem;
+.testimonial-section {
+  background: #F0F2F8;
 }
 
 .testimonial-section .testimonial-box {
-  border: 5px solid #0FCE7C;
+  border: 3px solid #3E00FF;
   border-radius: 5px;
 }
 
 .testimonial-section .testimonial-box  .img-container {
-  border: 6px solid #0FCE7C;
-  height: 4rem;
-  width: 4rem;
+  border: 2px solid #3E00FF;
+  height: 5rem;
+  width: 5rem;
+  top: -7px;
+  position: relative;
+  background: #fff;
 }
 
 .testimonial-section .testimonial-box .img-container.org-container {
   left: -1rem;
 }
 
+.testimonial-section .testimonial-box .img-container.org-container img {
+  left: -2px;
+  top: 6px;
+}
+
 .testimonial-section .quote-container {
   min-height: 9.8rem;
 }
 
 .testimonial-section .quote-text {
-  font-size: 1.3rem;
+  font-size: 1.2rem;
+}
+
+.testimonial-section img {
+  max-width: 5rem;
+  top: -5px;
+  background: white;
 }
 
 .testimonial-section .testimonial-photo-container {
   top: -2rem;
   left: 0;
   right: 0;
+}
+
+#pro,
```

Comment:
why not just 
```
#pro {
    color:white
    background-color: #170640;
}  
```  
or with a class always is better.

Reply:
^ Cause the h2 + others have have a colo black assigned to them in plans.css 
This is just for users and hence the reason for writing it up